### PR TITLE
Added dorothea/progeny

### PR DESCRIPTION
**Pull request by PauBadiaM**
Added links to dorothea/progeny in the ecosystem documentation

### CODE REVIEW

**Comment by ivirshup** [comment]
Thanks for the PR! I think "functional tools" may be a little generic/ ambiguous with the programming concept. Is there another term that could be used here?

**Comment by PauBadiaM** [comment]
I changed the name to "Footprint-based enrichment analysis", does it fit better?

**Comment by ivirshup** [comment]
Would "Annotation/ Enrichment Analysis" fit?

**Comment by PauBadiaM** [comment]
It is an enrichment analysis but foot-print based: we don't just look at the elements of a pathway/TF but also the biological downstream effects that occur when said biological process is active. 
"Annotation/ Enrichment Analysis" fits but it would be good to also mention somewhere that they are foot-print based. Would this be okay?

**Comment by LuckyMD** [comment]
Would it be sufficient to put the "footprint-based" in the function description? I guess it would be helpful to have a category of methods that will be easily recognizable to people if they're looking for a tool. That would also mean having a category that tools from multiple labs could fall under.

**Comment by PauBadiaM** [comment]
Yes that would work too :+1:

File: docs/ecosystem.rst [hunk 1]
```diff
@@ -81,3 +81,12 @@ Adaptive immune receptor repertoire (AIRR)
 Feature selection
 -----------------
 * `triku 🦔 <https://gitlab.com/alexmascension/triku>`__ :small:`Biodonostia Health Research Institute`
+
+
+Footprint-based enrichment analysis
+----------------
+Extract functional signatures based on curated prior knowledge
```
Comment (ivirshup) [inline docs/ecosystem.rst]:
```suggestion
Annotation/ Enrichment Analysis
-------------------------------

Analyses using curated prior knowledge
```

File: docs/ecosystem.rst [hunk 2]
```diff
@@ -81,3 +81,12 @@ Adaptive immune receptor repertoire (AIRR)
...
+
+* `dorothea <https://github.com/saezlab/dorothea-py>`__ for Transcription Factor activities :small:`Institute for Computational Biomedicine, Heidelberg University`
+* `progeny <https://github.com/saezlab/progeny-py>`__ for pathway activities :small:`Institute for Computational Biomedicine, Heidelberg University`
```
Comment (ivirshup) [inline docs/ecosystem.rst]:
```suggestion
* `progeny <https://github.com/saezlab/progeny-py>`__ for footprint-based pathway activities :small:`Institute for Computational Biomedicine, Heidelberg University`
```

**Review by ivirshup — COMMENTED** [review]
I've added some suggestions, do these look good?

File: docs/ecosystem.rst (hunk 2 above)
Comment (PauBadiaM) [reply docs/ecosystem.rst]: Yes, perfect. Thanks!

**Pull request merged**